src/managers/input/InputManager.cpp

```diff
@@ -728,10 +743,23 @@ void CInputManager::setMouseConfigs() {
             }
 
             const auto LIBINPUTSENS = std::clamp((HASCONFIG ? g_pConfigManager->getDeviceFloat(devname, "sensitivity") : g_pConfigManager->getFloat("input:sensitivity")), -1.f, 1.f);
-
-            libinput_device_config_accel_set_profile(LIBINPUTDEV, LIBINPUT_CONFIG_ACCEL_PROFILE_NONE);
             libinput_device_config_accel_set_speed(LIBINPUTDEV, LIBINPUTSENS);
 
+            const auto ACCELPROFILE = HASCONFIG ? g_pConfigManager->getDeviceString(devname, "accel_profile") : g_pConfigManager->getString("input:accel_profile");
+
+            if (ACCELPROFILE != "" || ACCELPROFILE != STRVAL_EMPTY) {
```

Comment:
this is always false idiot

Reply:
my bad I copied it from the final `else if` I had before restructuring.